src/commands/fun/rollDice.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable prettier/prettier */
+import { Message } from 'discord.js';
+import { ApplyOptions } from '@sapphire/decorators';
+import { Args, Command, CommandOptions } from '@sapphire/framework';
+import { BOT_PREFIX } from '../../bot'
+
+@ApplyOptions<CommandOptions>({
+      name: 'roll-dice',
+      aliases: ['rd', 'roll', 'rolldice', 'dice-roll', 'diceroll', 'dice'],
+      fullCategory: ['fun'],
+      description: 'Roll a dice!',
+      detailedDescription: `Example: ${BOT_PREFIX} roll-dice <size>`
+})
+class RollDiceCommand extends Command {
+  getRandomInt(max: number): number {
+    return Math.floor(Math.random() * max) + 1;
+  }
+
+  async messageRun(message: Message, args: Args): Promise<Message> {
+    const sides = await args.pick('number').catch(() => 6);
```

Comment:
Nice catch! Regarding this do you mean checking the number of arguments so that it if there are any extra arguments that we should check for that and handle that accordingly or do you mean type checking. Also what exactly do you mean by command prompts?

Reply:
Yup, I meant both extra arguments as well as type checking. The following is an example of command prompts from Commando.
![image](https://user-images.githubusercontent.com/58180935/157527451-b84ce38a-a1d8-45f6-b72b-a806e4ad4f1e.png)
Command prompts allow a user to type in arguments one at a time instead of all at once on the same line.

Reply:
sounds good added args and type checking for both